automation/src/test/scala/org/broadinstitute/dsde/firecloud/auth/UserAuthToken.scala

```diff
@@ -0,0 +1,25 @@
+package org.broadinstitute.dsde.firecloud.auth
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
+import org.broadinstitute.dsde.firecloud.config.{Config, Credentials}
+
+import scala.collection.JavaConverters._
+
+// Note: we are creating a new auth token from google every time we call this case class
```

Comment:
This comment isn't really true is it? Looks like it's using an existing pem file.

Reply:
`AuthToken.makeToken()` calls `cred.refreshToken()` - so maybe the comment belongs with `AuthToken` instead.